src/test/counterComm.test.js

```diff
@@ -0,0 +1,21 @@
+import counterModule from '../modules/counterComm.js';
+
+describe('Test comment counter', () => {
+  test('Should return 0 when recibe message "whitout comments"', () => {
+    const message = 'whitout comments';
+    const result = counterModule(message);
+    expect(result).toBe(0);
+  });
```

Comment:
- [ ] Good Job adding the tests team, But please add a test for the like counter. 
- [ ] Also, we need to have tests for all edge cases. This includes, `0`, `1`, and >`1` comments, like, or items, Please apply this test for all counter functions.

Reply:
We create a test for all conter functions

Reply:
solved